mixins/localize-static-mixin.js

```diff
@@ -0,0 +1,20 @@
+import { LocalizeMixin } from './localize-mixin.js';
+
+export const LocalizeStaticMixin = superclass => class extends LocalizeMixin(superclass) {
+
```

Comment:
Should this define the static resources property and default it to return `{ 'en': {} }` (to avoid explosions)?

Reply:
Yeah good point... it will indeed explode otherwise.